resources/assets/js/stories/StatusIcon.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { text } from "@storybook/addon-knobs";
+import StatusIcon, { IconStatus } from "../components/StatusIcon";
+
+const stories = storiesOf("Components|Status Icon", module);
+
+stories
+  .add(
+    "Default",
+    (): React.ReactElement => (
+      <div data-c-grid="gutter" data-c-padding="left(double)">
+        <StatusIcon size={text("Size", "h4")} status={IconStatus.DEFAULT} />
+      </div>
+    ),
+  )
+  .add(
+    "Error",
+    (): React.ReactElement => (
+      <div data-c-grid="gutter" data-c-padding="left(double)">
+        <StatusIcon size={text("Size", "small")} status={IconStatus.ERROR} />
+      </div>
+    ),
+  )
+  .add(
+    "Custom Color",
+    (): React.ReactElement => (
+      <div data-c-grid="gutter" data-c-padding="left(double)">
```

Comment:
Awkwardly, the top bar of the Storybook ui covers most of the icon. Do you get this too? 😆 
![image](https://user-images.githubusercontent.com/4313717/88709621-6e020080-d0e3-11ea-8157-3051531ffa5b.png)

Reply:
I do indeed, forgot to put a wrapping container on them